### DISCUSSION TITLE
Switch between tabs without refreshing whole page

### DISCUSSION THREAD

**Discussion by Wojtek17** [apache/superset#24929] [Ideas]
Hey, currently superset refreshes whole page when there is embeeded link within some element to a tab. Is there any possible way that it would show the tab without refreshing whole page? It doesn't make much sense and takes too much time. 
![image](https://github.com/apache/superset/assets/112469140/0937a99a-4e90-489c-845d-fabf4f18ca64)

**Comment by mohittyagi82** [comment]
Hi, Currently Superset refreshes whole page when there is embedded link within some element to a tab. Is there any possible way that it would show the tab without refreshing whole page? It doesn't make much sense and takes too much time.

Thanks
Mohit Tyagi
